Core/include/Acts/Utilities/MathHelpers.hpp

```diff
@@ -12,9 +12,43 @@
 
 namespace Acts {
 
+/// @brief Calculates the ordinary power of the number x.
+/// @param x: Number to take the power from
+/// @param p: Power to take
+template <typename T, typename P>
+constexpr T pow(T x, P p)
+  requires(std::is_integral_v<P>)
```

Comment:
```suggestion
/// @brief Calculates the ordinary power of the number x.
/// @param x: Number to take the power from
/// @param p: Power to take
template <typename T, typename P>
constexpr T pow(T x, P p)
  requires(std::is_integral_v<P>)
```
- I think `typename P` can be replaced by something like `std::integral P`? then the `requires` can be dropped
- for integer exponents this is usually done with https://en.wikipedia.org/wiki/Exponentiation_by_squaring - not sure if relevant